yolapy/resources/site.py

```diff
@@ -69,22 +63,22 @@ def change_site_domain(self, site_id, new_domain):
     def delete_site(self, site_id):
         """Delete a site.
 
-        ```
-        yola.get_site('site_id')['deleted_at'] # => None
-        yola.delete_site('site_id')
-        yola.get_site('site_id')['deleted_at'] # => timestamp
-        ```
+        >>> yola.get_site('site_id')['deleted_at']
+        None
+        >>> yola.delete_site('site_id')
+        >>> yola.get_site('site_id')['deleted_at']
+        '2015-08-27T10:00:07'
         """
         self.delete(self._site_path(site_id))
 
     def undelete_site(self, site_id):
         """Un-delete a site.
 
-        ```
-        yola.get_site('site_id')['deleted_at'] # => timestamp
-        yola.delete_site('site_id')
-        yola.get_site('site_id')['deleted_at'] # => None
-        ```
+        >>> yola.get_site('site_id')['deleted_at']
+        '2015-08-27T10:00:07'
+        >>> yola.delete_site('site_id')
```

Comment:
Fixed in https://github.com/yola/yolapy/commit/b675a76c246d06693809d99f618585a498b11278